nucliadb/src/nucliadb/ingest/orm/brain.py

```diff
@@ -232,15 +233,22 @@ def apply_field_metadata(
             full_field_id = ids.FieldId(rid=self.rid, type=field_type, key=field_name).full()
             self.brain.paragraphs_to_delete.append(full_field_id)
 
+        field_relations = self.brain.field_relations[field_key].relations
         for relations in metadata.metadata.relations:
             for relation in relations.relations:
                 self.brain.relations.append(relation)
 
+                index_relation = IndexRelation(relation=relation)
+                if relation.metadata.HasField("data_augmentation_task_id"):
+                    index_relation.facets.append("/g/da/{relation.metadata.data_augmentation_task_id}")
```

Comment:
Missing f-string
```suggestion
                    index_relation.facets.append(f"/g/da/{relation.metadata.data_augmentation_task_id}")
```

Reply:
Good catch. I guess we would have seen it when adding tests, but better to fix now.